bin/oneoff/gh-ost_migrations/add_script_id_deleted_at_columns_to_channel_tokens.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+DB_USER=${DB_USER?Required}
+DB_PASSWORD=${DB_PASSWORD?Required}
+DB_MASTER_HOST=${DB_MASTER_HOST?Required}
+
+# Options reference: https://github.com/github/gh-ost/blob/master/go/cmd/gh-ost/main.go
+args=(
+  # MySQL user
+  --user=${DB_USER}
+
+  # MySQL password
+  --password=${DB_PASSWORD}
+
+  # We don't have a reporting replica anymore, so carry out all operations on primary database
+  --allow-on-master
+
+  # MySQL hostname (preferably a replica, not the master)
+  --host=${DB_MASTER_HOST}
+
+  # (optional) explicitly tell gh-ost the identity of the master.
+  # Format: some.host.com[:port]
+  # This is useful in master-master setups where you wish to pick an explicit master,
+  # or in a tungsten-replicator where gh-ost is unable to determine the master
+  --assume-master-host=${DB_MASTER_HOST}
+
+  # while this file exists, migration will postpone the final stage of swapping tables,
+  # and will keep on syncing the ghost table.
+  # Cut-over/swapping would be ready to perform the moment the file is deleted.
+  --postpone-cut-over-flag-file=/tmp/gh-ost.cutover
+
+  # directory where hook files are found (default: empty, ie. hooks disabled).
+  # Hook files found on this path, and conforming to hook naming conventions will be executed
+  --hooks-path=${PWD}
+
+  # database name (mandatory)
+  --database="dashboard_production"
+
+  # table name (mandatory)
+  --table="channel_tokens"
+
+  # verbose
+  --verbose
+
+  # alter statement (mandatory)
+  --alter="ADD `script_id` int, ADD `deleted_at` datetime DEFAULT NULL, DROP INDEX `index_channel_tokens_on_storage_id_and_level_id`, ADD UNIQUE `index_channel_tokens_on_storage_level_script_ids_and_deleted_at` (`storage_id`, `level_id`, `script_id`, `deleted_at`)"
```

Comment:
Does this make both script_id and deleted_at default to null? Presume you've run this bit of SQL somewhere?

Reply:
I think only `deleted_at` would be DEFAULT NULL according to the [`ALTER TABLE` documentation](https://dev.mysql.com/doc/refman/5.7/en/alter-table.html):

> Multiple ADD, ALTER, DROP, and CHANGE clauses are permitted in a single ALTER TABLE statement, separated by commas

So the `ADD 'script_id' int` clause is separated from the `ADD 'deleted_at' datetime DEFAULT NULL` clause.